data/xcspecs.bzl

```diff
@@ -151,24 +151,27 @@ SETTINGS = {
                     "gnu++14",
                     "c++17",
                     "gnu++17",
+                    "c++20",
+                    "gnu++20",
                     "compiler-default",
                 ],
             },
             "CLANG_CXX_LIBRARY": {
                 "AdditionalLinkerArgs": {
                     "<<otherwise>>": ["-stdlib=$(value)"],
                     "compiler-default": [],
+                    "libstdc++": [],
                 },
                 "AppearsAfter": "CLANG_CXX_LANGUAGE_STANDARD",
-                "Category": "LanguageCXX",
                 "CommandLineArgs": {
                     "<<otherwise>>": ["-stdlib=$(value)"],
                     "compiler-default": [],
+                    "libstdc++": [],
                 },
                 "DefaultValue": "com_apple_compilers_llvm_clang_1_0__CLANG_CXX_LIBRARY__DefaultValue",
                 "FileTypes": ["sourcecode.cpp.cpp", "sourcecode.cpp.objcpp"],
                 "Type": "Enumeration",
-                "Values": ["libstdc++", "libc++", "compiler-default"],
```

Comment:
Does this cause a value error if someone still uses `"libstdc++"` for Xcode 12.5?